types/ramda/test/anyPass-tests.ts

```diff
@@ -20,4 +20,29 @@ import * as R from 'ramda';
     f(8);
     // $ExpectType boolean
     f(9); // => false
+
+    const isFooAndBar = (a: {}): a is { foo: number; bar: string } => true;
```

Comment:
In the unit test, it's better to give one proper implementation of every type guard function, instead of directly return a `true` value, such as 

```ts
const isFooAndBar = (a: {}): a is { foo: number; bar: string } => {
    return a.foo != undefined && a.bar != undefined;
};
```